### ISSUE TITLE
[1.0pr2_1] GuiNode#containerAcceptsKey crash

### ISSUE THREAD

**Issue by PalePyth** [LambdaInnovation/AcademyCraft#235]
https://bpaste.net/show/c1ad298f9ad1
Could you fix it?

**Comment by WeAthFoLD** [comment]
Will look into it. For self reference the important part is pasted here:

```
java.lang.NullPointerException: Updating screen events
	at cn.academy.energy.client.gui.node.GuiNode.containerAcceptsKey(GuiNode.java:94)
	at cn.liutils.cgui.gui.LIGuiContainer.func_73869_a(LIGuiContainer.java:86)
	at net.minecraft.client.gui.inventory.GuiContainer.public_func_73869_a(GuiContainer.java)
	at codechicken.nei.guihook.GuiContainerManager.callKeyTyped(GuiContainerManager.java)
	at codechicken.nei.guihook.GuiContainerManager.keyTyped(GuiContainerManager.java:475)
	at codechicken.nei.guihook.GuiContainerManager.handleKeyboardInput(GuiContainerManager.java:465)
	at net.minecraft.client.gui.inventory.GuiContainer.func_146282_l(GuiContainer.java)
	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:276)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1640)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by PalePyth** [comment]
I have no idea also I don't know ho to code :D

**Comment by PalePyth** [comment]
Could you tell me how to fix it please?

**Comment by WeAthFoLD** [comment]
@PalePyth It must be fixed through code. I will fix it in next release, but have to wait for a week or so because I'm having end-term exam >_>

**Comment by PalePyth** [comment]
Good luck in your exam!

**Comment by WeAthFoLD** [comment]
@PalePyth What did you do to make this crash happen? Can the crash be stably reproduced (always happens)?

**Comment by Renniuq** [comment]
I got a similar crash (complaining about rendering the successful linking of a node to a matrix) after I had moved the node with a frame from funkylocomotion. It was repeated the next time I logged in and linked a new node, but after than it worked fine again and ceased to cause crashes when I linked more nodes, or old ones. Looking at the logs, they are both in fact identical to the one posted above (at least to the "important part" you posted above).
